src/app/management/commands/update-stats.py

```diff
@@ -23,6 +23,7 @@ def handle(self, **options):
             delegate_response = requests.get(delegate_url.format(delegate.public_key))
             voters_response = requests.get(voters_url.format(delegate.public_key))
             if delegate_response.status_code != 200 or voters_response.status_code != 200:
+                self.stderr.write(f'Was not able to get delegate data for {delegate.name}')
```

Comment:
should it be `stdout` instead?

Reply:
nope.
`self.stdout` prints out text in default terminal colour, while `self.stderr` prints it out in error (usually red) color. In this case, this is an error and should attract your attention :)